### DISCUSSION TITLE
Is it possible to render table value with style ?

### DISCUSSION THREAD

**Discussion by yellowbean** [reflex-dev/reflex#3020] [Q&A]
Great library work ! Thank you guys

I'm trying to render data using table component : https://reflex.dev/docs/library/datadisplay/table/

The goal is to render a string with different color in the parenthesis .

![image](https://github.com/reflex-dev/reflex/assets/1008321/7a65bac3-cf7f-4fda-8d0b-44c2daf61ff2)

If above highlight doesn't work, is there a way to render cells with style like below ? 

![image](https://github.com/reflex-dev/reflex/assets/1008321/ed6a7289-77b1-498e-8400-154ba646337d)

**Comment by picklelo** [comment]
Within each table cell you should be able to set the color:

```python
rx.table.cell("100", rx.text.span("(10)", color="red"))
```
